libnd4j/tests_cpu/layers_tests/DeclarableOpsTests10.cpp

```diff
@@ -1528,6 +1551,105 @@ TEST_F(DeclarableOpsTests10, ImageResizeBilinear_Test01) {
 
 }
 
+TEST_F(DeclarableOpsTests10, ResizeImages_Test1) {
+
+    NDArray input = NDArrayFactory::create<float>('c', {2, 4, 5, 3});
+    input.linspace(1.);
+
+    auto expected = NDArrayFactory::create<float>('c', {2, 7, 9, 3}, {
+            1., 2., 3., 2.6666667, 3.6666667,
```

Comment:
Either make this array double, or add f literal. MSVC will be very disappointed otherwise.

Reply:
Fixed to avoid MSVC compilation issues.